easybuild/easyconfigs/r/ReFrame/ReFrame-3.12.0.eb

```diff
@@ -0,0 +1,67 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'PythonBundle'
+
+name = 'ReFrame'
+version = '3.12.0'
+
+homepage = 'https://github.com/reframe-hpc/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+req_py_majver = 3
+req_py_minver = 6
+
+use_pip = True
+
+exts_list = [
+    ('pip', '21.3.1', {
```

Comment:
Please add back the comment that explain why we're not using latest version of `pip`, see also https://github.com/easybuilders/easybuild-easyconfigs/blob/f590f2791d3dc5a26c7590ac11b794633e0cc18a/easybuild/easyconfigs/r/ReFrame/ReFrame-3.11.2.eb#L22

Reply:
I blame inject-checksum for the missing comment...